ec2utils/ec2uploadimg/lib/ec2utils/ec2uploadimg.py

```diff
@@ -238,6 +269,8 @@ def _clean_up(self):
             for volume in self.created_volumes:
                 self._detach_volume(volume, True)
                 self._remove_volume(volume)
+        if self.temporary_security_group_id:
```

Comment:
Maybe we need a wait here till the instance is deregisted?
Anyway, I'm not sure if we want to delete the group anyway as for instance the launch-wizard also does not delete the group. @rjschwei what do you think?

Reply:
I added now a waiter because otherwise it will fail with a dependency error!